plugins/portals-list.js

```diff
@@ -257,9 +305,19 @@ window.plugin.portalslist.portalTable = function(sortBy, sortOrder, filter) {
 
   if(filter !== 0) {
     portals = portals.filter(function(obj) {
-      return filter < 0
-        ? obj.portal.options.team+1 != -filter
-        : obj.portal.options.team+1 == filter;
+      if (Math.abs(filter) <= 3) {
+        return filter < 0
+          ? obj.portal.options.team+1 != -filter
```

Comment:
eqeqeq

Reply:
@le-jeu ?

Reply:
see https://github.com/IITC-CE/ingress-intel-total-conversion/pull/443/files/4b3b39bfe785ea5145cd618d48ed315a60e3390a#r586615913

Reply:
@le-jeu 
What do you mean? (May be url is broken)